quickstarts/maven/micronaut4/pom.xml

```diff
@@ -0,0 +1,189 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2019 Red Hat, Inc.
+    This program and the accompanying materials are made
+    available under the terms of the Eclipse Public License 2.0
+    which is available at:
+
+        https://www.eclipse.org/legal/epl-2.0/
+
+    SPDX-License-Identifier: EPL-2.0
+
+    Contributors:
+      Red Hat, Inc. - initial API and implementation
+
+-->
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+	<groupId>io.micronaut.platform</groupId>
+  	<artifactId>micronaut-parent</artifactId>
+	<version>4.1.0</version>
+  </parent>
+
+  <groupId>org.eclipse.jkube.quickstarts.maven</groupId>
+  <artifactId>micronaut</artifactId>
+  <version>1.16.1</version>
+  <name>Eclipse JKube :: Quickstarts :: Maven :: Micronaut 4</name>
+  <description>
+    Micronaut application featuring REST endpoints (micronaut-http) with validation (micronaut-validation).
+  </description>
+
+  <properties>
+	<jdk.version>17</jdk.version>
+    <release.version>17</release.version>
+    <jkube.version>${project.version}</jkube.version>
+    <micronaut.version>${project.parent.version}</micronaut.version>
+    <exec.mainClass>org.eclipse.jkube.quickstart.micronaut4.custom.Application</exec.mainClass>
+    <jkube.enricher.jkube-service.type>NodePort</jkube.enricher.jkube-service.type>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.micronaut</groupId>
+      <artifactId>micronaut-jackson-databind</artifactId>
+      <scope>compile</scope>
+    </dependency>
+	<dependency>
+	  <groupId>io.micronaut.testresources</groupId>
+	  <artifactId>micronaut-test-resources-client</artifactId>
+	  <scope>provided</scope>
+	</dependency>
+	<dependency>
+      <groupId>io.micronaut.test</groupId>
```

Comment:
nit, indentation seems a little off in this file. I see at some places you're using tabs, some places you're using spaces.
Please use spaces everywhere.

Reply:
Sure!